packages/components/src/keyboard-shortcuts/index.js

```diff
@@ -21,7 +21,22 @@ class KeyboardShortcuts extends Component {
 		const { keyTarget = document } = this;
 
 		this.mousetrap = new Mousetrap( keyTarget );
+
 		forEach( this.props.shortcuts, ( callback, key ) => {
+			if ( process.env.NODE_ENV === 'development' ) {
+				const keys = key.split( '+' );
+				const modifiers = new Set( keys.filter( ( value ) => value.length > 1 ) );
```

Comment:
Won't this capture much more than just modifiers, like `escape`, or `space`, or `tab` ? While ultimately it has no impact one way or the other since the subsequent lines only care to check for `alt` or `shift` specifically, it begs the question at that point why to filter at all (vs. `hasAlt = keys.includes( 'alt' )`).